app/src/main/res/layout-land/question_player_fragment.xml

```diff
@@ -116,36 +143,6 @@
         tools:layout_constraintGuide_percent="0.5f" />
     </androidx.constraintlayout.widget.ConstraintLayout>
 
-    <FrameLayout
```

Comment:
Why was this removed? We need this.

Reply:
@aishapervin Apologies for the confusion here, I thought it isn't needed here. Thanks, @rt4914

Reply:
done